src/ServiceProvider.php

```diff
@@ -15,9 +18,9 @@ class ServiceProvider extends IlluminateServiceProvider
     public function register()
     {
         $this->mergeConfigFrom($this->getConfigPath(), 'unleash');
+        $this->app ->when(Unleash::class)->needs(ClientInterface::class)->give(Client::class);
```

Comment:
There looks to be a rouge space after `app`, is this needed?

Reply:
Sharp eyes! I have pushed an update.